packages/firestore/src/api/user_data_reader.ts

```diff
@@ -314,14 +320,16 @@ export class UserDataReader {
     methodName: string,
     targetDoc: DocumentKey,
     input: unknown,
-    options: firestore.SetOptions = {}
+    options: firestore.SetOptions = {},
+    hasConverter: boolean
```

Comment:
Nit: Flip argument order to make optional argument go last.

Reply:
done.